### PR TITLE
initial css structure

### LINKED ISSUE #3: Create page structure styles
Add styling to match the grid of the original VIP Area page:
https://www.realmadrid.com/area-vip/partido-vip

### PR DESCRIPTION
**Pull request by msagolj**
Inital css structure , for mobile, desktop and wide desktop.

Fix #3

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/
- After: https://issue-3--realmadrid--hlxsites.hlx.page/

### CODE REVIEW

**Comment by solaris007** [comment]
should probably establish early on variables for font-weights, font-sizes, colors, etc.

**Review by chicharr — CHANGES_REQUESTED** [review]
Hi @msagolj 
could you please check the padding /margin of the main content for mobile? it doesn't match the one from the original website.

Screenshot 2023-04-25 at 11 58 54

Also please configure your IDE to remove trailing spaces and use 2 space indentation, instead of 4. If you are using Visual Studio Code I can help you with that

**Comment by msagolj** [comment]
@chicharr

> could you please check the padding /margin of the main content for mobile? it doesn't match the one from the original website.

 I didn't add any padding/margin in a first step as they define the margins/paddings on each  div container e.g:
 
image
and they are not even the same everywhere :) 

> Also please configure your IDE to remove trailing spaces and use 2 space indentation, instead of 4.
Sorry forgot to run the npm install and install the lint extensions for this workspace. should be fixed with next commit.

**Review by chicharr — APPROVED** [review]

File: styles/styles.css [hunk 1]
```diff
@@ -26,118 +26,108 @@
...
-  font-weight: 600;
+  font-weight: 500;
   line-height: 1.25;
-  margin-top: 1em;
-  margin-bottom: .5em;
-  scroll-margin: calc(var(--nav-height) + 1em);
+  margin-bottom: 15px;
+  padding-bottom: 5px;
+  color: #333;
+  border-bottom: 4px solid #0255A5;
```
Comment (vdua) [inline styles/styles.css:116]: We are using h2 in the hero block as well and this adds a border there as well. So I think we need to update the [styling of the h2](https://github.com/hlxsites/realmadrid/blob/main/blocks/hero/hero.css#L46) in the hero block there as well. 

For the vip areas section this is the styling that works

```
main .section .default-content-wrapper h2 {
  margin: 0;
  padding: 0;
  width: 40%;
  margin-bottom: 21px; /** for desktop and 15px for mobile */
  border-left: 0;
  text-align: left;
  border-right: 4px solid transparent; 
  text-transform: uppercase;
  font-family: var(--heading-font-family);
  font-size: 20px; /* for desktop this variable is not defined */
  font-weight: 500;
  color: var(--heading-color); /*333*/
  padding-bottom: 5px; 
  border-bottom: 4px solid var(--heading-border-color);
}
```

Comment (chicharr) [reply styles/styles.css:116]: Thanks @vdua I will address your comments in the follow-up ticket for the Hero component

**Pull request merged**